src/pages/home/report/ReportActionsView.js

```diff
@@ -39,6 +37,9 @@ const propTypes = {
     report: PropTypes.shape({
         // Number of actions unread
         unreadActionCount: PropTypes.number,
+
+        // Number of the last sequence
+        maxSequenceNumber: PropTypes.number,
```

Comment:
Can't we always figure this out if we are able to access `reportActions`?

Reply:
Maybe with `sortedReportActions`? But I'm not really adding anything new here. The report is already being passed and it already has the `maxSequenceNumber` in it. React just complained if I accessed it without declaring it inside the shape, so it is fine this way, right?

Reply:
Looks unused now?